src/components/index.d.ts

```diff
@@ -936,6 +960,7 @@ export declare const DateInput: {
 }
 
 import * as Validators from '../validators/BirthdateInputValidator'
+import { Node } from '@babel/core'
```

Comment:
what is this for?

Reply:
VSC must have auto-imported that when I was selecting `Node` on line 639 (which I replaced with `React.ReactNode`); good catch